gst/tensor_decoder/tensordec.c

```diff
@@ -717,7 +767,8 @@ gst_tensordec_get_label (GstTensorDec * self,
       gst_memory_unmap (mem, &info);
     }
   }
-
+  self->tensordec_image_label.image_label = gst_get_image_label (self);
```

Comment:
```for (i = 0; i < num_mems; i++)```: #586 Very strange. If it's multi-tensor, only one of them will be valid for image-labels.

Reply:
I will consider it for next step